### PR TITLE
update from db

### PR DESCRIPTION
**Pull request by ozym**
@elidana @samgeonet update from the db, has been a little while since the last one. Feel free to  nominate alternate reviewers ...

### CODE REVIEW

File: install/antennas.csv [hunk 1]
```diff
@@ -328,7 +328,7 @@ Trimble Navigation Ltd.,TRM57971.00,4127561886,RGHR,0.002,0,0,0,2016-08-15T23:43
 Trimble Navigation Ltd.,TRM57971.00,4127561893,VGOB,0.055,0,0,0,2016-05-12T01:01:58Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561896,PAWA,0.055,0,0,0,2016-03-01T02:36:52Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561900,LRR1,0.002,0,0,0,2016-11-15T04:46:08Z,2017-05-22T00:14:00Z
-Trimble Navigation Ltd.,TRM57971.00,4127561900,GLOK,0.002,0,0,0,2017-06-12T04:25:09Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM57971.00,4127561900,GLOK,0.002,0,0,0,2017-06-12T04:25:09Z,2017-09-20T03:24:31Z
```
Comment (elidana) [inline install/antennas.csv]: Hi @ozym , this is a wrong entry on the db that we are going to fix this morning. 
(all latest entries related to LOK1-GLOK will need to be corrected)

File: install/receivers.csv [hunk 2]
```diff
@@ -224,7 +224,7 @@ Trimble Navigation Ltd.,TRIMBLE NETR9,5504R50170,LOK1,2016-11-15T21:44:30Z,9999-
 Trimble Navigation Ltd.,TRIMBLE NETR9,5505R50001,KERE,2016-08-18T00:55:32Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRIMBLE NETR9,5505R50003,BTHL,2016-12-14T22:34:22Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRIMBLE NETR9,5505R50016,LRR1,2016-11-15T04:46:08Z,2017-05-22T00:14:00Z
-Trimble Navigation Ltd.,TRIMBLE NETR9,5505R50016,GLOK,2017-06-12T04:25:09Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRIMBLE NETR9,5505R50016,GLOK,2017-06-12T04:25:09Z,2017-09-20T03:24:31Z
```
Comment (elidana) [inline install/receivers.csv]: another wrong entry for GLOK that will be fixed this morning

File: install/sessions.csv [hunk 3]
```diff
@@ -143,7 +144,7 @@ GISB,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2015-08-24T15:
 GLDB,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,linz,trimble_5700 x4,2004-02-02T00:00:00Z,2011-01-25T06:20:00Z
 GLDB,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,3,linz,trimble_netr9 x4,2011-01-25T07:11:01Z,2015-09-06T16:25:00Z
 GLDB,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2015-09-06T16:25:01Z,9999-01-01T00:00:00Z
-GLOK,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2017-06-12T04:25:09Z,9999-01-01T00:00:00Z
+GLOK,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2017-06-12T04:25:09Z,2017-09-20T03:24:31Z
```
Comment (elidana) [inline install/sessions.csv]: another wrong entry

**Comment by ozym** [comment]
@elidana I can wait easily until you update the details and make another push to this pull request, and then merge.

**Comment by elidana** [comment]
Thanks @ozym , that would be awesome. 
we're collecting the info now, will let you know when it's done. (and depending on the complexity I might ask for some help to change the Oracle-db!)

**Comment by elidana** [comment]
Hi @ozym , GLOK-LOK1 is now fixed (hopefully).
There's another wrong entry that I need to fix for TEMA, but was inserted after this pr. If you want to wait until I fix that one too, I can let you know when this is done.

**Comment by elidana** [comment]
all done now! thanks to @dc735 ! 
@ozym , can you please rerun the update now? Thanks!

**Comment by ozym** [comment]
@elidana rerun the update.

File: install/antennas.csv [hunk 4]
```diff
@@ -335,7 +336,7 @@ Trimble Navigation Ltd.,TRM57971.00,4127561911,RGHL,0.002,0,0,0,2017-06-08T01:30
 Trimble Navigation Ltd.,TRM57971.00,4127561914,WPAW,0.002,0,0,0,2016-08-15T02:08:08Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561917,VGPK,0.055,0,0,0,2016-08-23T01:10:33Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561919,TGWH,0.002,0,0,0,2016-10-12T03:04:20Z,9999-01-01T00:00:00Z
-Trimble Navigation Ltd.,TRM57971.00,4127561923,LOK1,0,0,0,0,2016-11-15T21:44:30Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM57971.00,4127561923,LOK1,0,0,0,0,2016-11-15T21:44:30Z,2017-09-20T03:24:30Z
```
Comment (elidana) [inline install/antennas.csv]: this one (and GDS1 at line 348) are a bit weird.
Both sites have their receiver swapped on `2017-09-20`. They're both offline, so we don't know the exact timestamp to start the cgps-session of the latest receiver. Both sites still have the same antenna and, looking at _magma_ web interface, the antenna end dates are "open".

**Pull request merged**